oaknational/Oak-Web-Application
# LESQ-144 Feat/new worksheeet style

## Description

remove expanding container
add brush borders around worksheet

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1782--oak-web-application.netlify.app//beta/teachers/programmes/biology-secondary-ks4/units/cells-biology-3p8njnj/lessons/prokaryotic-and-eukaryotic-cells-b9qqye
3. You should see newly styled worksheet

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
